grafana/scylla-dash-per-server.master.json

```diff
@@ -1751,6 +1768,7 @@
                                 "step": 1
                             }
                         ],
+                        "description": "Holds a number of currently pending allocations. A non-zero value indicates that we have a bottleneck in the disk write flow",
```

Comment:
holds the, and I think you should remove all the holds.
Have you check that the long description looks ok?